src/components/TimelineComponent.tsx

```diff
@@ -713,8 +737,8 @@ export default class TimelineComponent
     const axisLayer = this.svg.append('g')
         .attr('id', 'axisLayer');
 
-
-    console.log(ViewType[this.m.view] === ViewType.EventMagnitude);
+    this.debugConfig.consoleLogger(this.drawTimeline.name,
+        ViewType[this.m.view] === ViewType.EventMagnitude, 'INFO');
```

Comment:
The message should describe a bit what's happening here. Otherwise it'll just output "true" or "false"